src/ProgressiveColonizationSystem/GameData/ProgressiveColonizationSystem/Contracts/Rescues.cfg

```diff
@@ -0,0 +1,78 @@
+﻿
+CONTRACT_TYPE
+{
+    // See https://github.com/jrossignol/ContractConfigurator/wiki/Contract-Type
+    name = RescueAtJool
+    title = Rescue @/SpawnVessel/EscapePod/CREW/name from orbit of @/stuckAtBody and bring them to your base on @/goingToBody
+    genericTitle = Space Adventurer Rescue
+    contractGroup = SpaceAdventurers
+    description = @/SpawnVessel/EscapePod/CREW/name really wants to become a colonist on @/goingToBody.  Granted, @/SpawnVessel/EscapePod/CREW/name might not have all the skills, abilities, disposition, temperment, and all-around intellect that the kerbals KSP chose have, but @/SpawnVessel/EscapePod/CREW/name makes up for it with sheer goforit.  However, somewhere around @/stuckAtBody things took a turn for the worse (read: ran out of lawn chairs and balloons) and...  well...  a little help?
+    genericDescription = KSP goes through a highly selective vetting process for crewing stations, but sometimes kerbals feel like their amazing talents went unappreciated and set out for your colonies on their own.  Usually with mixed results.
+    synopsis = We want you to rescue a kerbal
+    completedMessage = @/SpawnVessel/EscapePod/CREW/name has joined your colony on @/goingToBody!
+    prestige = Significant
+
+    DATA
+    {
+        // This is where we're going to tote the kerbal to
+        type = CelestialBody
+        goingToBody = jool.Children().Random()
+    }
+    DATA
+    {
+        // This is where we're going to pick up the kerbal
+        type = CelestialBody
+        stuckAtBody = [Jool, Laythe, Vall, Tylo, Bop, Pol].Random()
+    }
+
+    PARAMETER
+    {
+        name = GetToNewHome
+        type = PksOnStationParameter
+
+        researchCategory = farming
+        tier = 3
+        
+        body = @/goingToBody
+        kerbal = @/SpawnVessel/EscapePod/CREW/name
+    }
+
+    REQUIREMENT
+    {
+        name = FarmingResearched
+        type = PksTier
+        researchCategory = farming
+        tier = 3
+        body = @/goingToBody
+    }
+
+	BEHAVIOUR
+	{
+		name = OrbitGenerator
+		type = OrbitGenerator
+		
+		RANDOM_ORBIT
+		{
+			targetBody = @/stuckAtBody
+			type = RANDOM
+		}
+	}
+
+    BEHAVIOUR
+    {
+        name = SpawnVessel
+        type = SpawnVessel
```

Comment:
This blows up because it wants an orbit.  I'd like it to use the OrbitGenerator, but I'm not sure that's even possible.